htdocs/product/class/api_products.class.php

```diff
@@ -1678,6 +1800,12 @@ private function _fetch($id, $ref = '', $ref_ext = '', $barcode = '', $includest
             throw new RestException(401, 'Access not allowed for login '.DolibarrApiAccess::$user->login);
         }
 
+		$prodcomb = new ProductCombination($this->db);
+        $this->product->fk_product_parent = null;
+        if (($fk_product_parent = $prodcomb->getFkProductParentByFkProductChild($this->product->id)) > 0) {
```

Comment:
This method seems to not exists.

Reply:
The method is added at commit 6bffbc4: https://github.com/Dolibarr/dolibarr/pull/14537/commits/6bffbc43f3f874e6a5c5894663a068f43aece767

Reply:
I did not find the commit you mention is an existing PR. Can you tell me in which PR i can find it ?